src/views/product-tutorials-view/helpers/index.ts

```diff
@@ -19,10 +19,8 @@ export function filterCollections(
 	)
 }
 
-export function sortAlphabetically(
-	property: keyof Pick<ClientCollection, 'shortName' | 'name'>
-) {
-	return (a: ClientCollection, b: ClientCollection) => {
+export function sortAlphabetically<A, B>(property: string) {
```

Comment:
Opening this function up to be more generic.

Reply:
Future thought 📌💭

Could probably be lifted to `src/lib`! <insert :eyebrows: Slackmoji here>